### DISCUSSION TITLE
how to compare over 100 different images?

### DISCUSSION THREAD

**Discussion by sitetwo** [romankh3/image-comparison#242] [Q&A]
Hi,
I'm trying to use ImageComparison class to compare images, but I have alot of images like over 100 image files, so does that mean I will have to create ImageComparison objects for every single comparison by using the statement below?  If that is the case, I will end up having over 100 objects.

``ImageComparison imageComparison = new ImageComparison(expectedImage, actualImage);``

**Comment by romankh3** [comment]
Hi,
By this time yes, we can do it only by creating object for each comparison.
If it's really needed feature, we can add it to next release, I don't see difficults with adding it.

Best regards,
@romankh3

**Reply by sitetwo** [reply]
Really? you're open to add new feature to the library?  that's pretty awesome! thank you.  But I'm trying to avoid to fix something that is not broken.  I think for now it's ok since I only create image objects and use within the methods so not too worry about wasting memory.  thank you
